### ISSUE TITLE
Class 'GrumPHP\Console\Application' not found

### ISSUE THREAD

**Issue by lasselehtinen** [phpro/grumphp#54]
I tried to do a global install on a Windows 7 64-bit machine, but I am getting the error mentioned in the topic.

```
$ composer global require phpro/grumphp
Changed current directory to C:/Users/lasleh/AppData/Roaming/Composer
Using version ^0.5.1 for phpro/grumphp
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing gitonomy/gitlib (v0.1.7)
    Downloading: 100%

  - Installing doctrine/collections (v1.3.0)
    Loading from cache

  - Installing symfony/config (v2.7.6)
    Loading from cache

  - Installing symfony/dependency-injection (v2.7.6)
    Loading from cache

  - Installing symfony/yaml (v2.7.6)
    Loading from cache

  - Installing symfony/expression-language (v2.7.6)
    Downloading: 100%

  - Installing phpro/grumphp (v0.5.1)
    Downloading: 100%

symfony/dependency-injection suggests installing symfony/proxy-manager-bridge (Generate service proxies to lazy load them)
phpro/grumphp suggests installing behat/behat (Lets GrumPHP validate your project features.)
phpro/grumphp suggests installing phpspec/phpspec (Lets GrumPHP spec your code.)
phpro/grumphp suggests installing phpunit/phpunit (Lets GrumPHP run your unit tests.)
phpro/grumphp suggests installing squizlabs/php_codesniffer (Lets GrumPHP sniff on your code.)
phpro/grumphp suggests installing roave/security-advisories (Lets GrumPHP be sure that there are no known security issues.) Writing lock file
Generating autoload files

Created git hooks folder at: ./.git/hooks/
Watch out! GrumPHP is sniffing your commits!

lasleh@CPB2265
$ grumphp
PHP Fatal error:  Class 'GrumPHP\Console\Application' not found in C:\Users\lasleh\AppData\Roaming\Composer\vendor\phpro\grumphp\bin\grumphp on line 31

Fatal error: Class 'GrumPHP\Console\Application' not found in C:\Users\lasleh\AppData\Roaming\Composer\vendor\phpro\grumphp\bin\grumphp on line 31
```

**Comment by veewee** [comment]
Hello @lasselehtinen,

There is a possible fix  that isn't released yet.
Can you try installing the master branch?

```
﻿﻿composer global require phpro/grumphp:dev-master
```

**Comment by lasselehtinen** [comment]
Now I get the following error if I run grumphp on a project where guzzle is already required in composer.json. Works fine on those without though.

```
grumphp
PHP Fatal error:  Cannot redeclare GuzzleHttp\uri_template() (previously declared in C:\path\to\folder\vendor\guzzlehttp\guzzle\src\functions.php:18) in C:\Users\lasleh\AppData\Roaming\Composer\vendor\guzzlehttp\guzzle\src\functions.php on line 265

Fatal error: Cannot redeclare GuzzleHttp\uri_template() (previously declared in C:\path\to\folder\vendor\guzzlehttp\guzzle\src\functions.php:18) in C:\Users\lasleh\AppData\Roaming\Composer\vendor\guzzlehttp\guzzle\src\functions.php on line 265
```

**Comment by veewee** [comment]
Do you have guzzle installed globally as well?
GrumPHP will try to load both the project autoloading as the system autoloading file.
Guzzle added this to the autoloader: `"files": ["src/functions_include.php"]`
This means that the functions are loaded twice and this triggers the fatal exception.

I don't think this is something we can do something about. If it's possible you can remove the globally installed guzzle?

**Comment by lasselehtinen** [comment]
I had couple global packages that had guzzle as required. Namely:

https://github.com/laravel/installer
https://github.com/laravel/lumen-installer
https://github.com/vinkla/climb (require knplabs/packagist-api which in turn requires guzzle)

I removed those and now it seems to work. Downside is of course that I cannot have these packages installed at the same time. So is this more of a bug in Guzzle / Composer? I found this but I am not smart enough to figure it out.

https://github.com/guzzle/guzzle/issues/676
